### PR TITLE
Apply some comments from LWG's Cologne review.

### PR DESCRIPTION
**Pull request by jyasskin**
Please apply these changes globally. We'll probably have more comments tomorrow, so no need to merge this today.

### CODE REVIEW

File: future.html [hunk 1]
```diff
@@ -143,15 +143,15 @@
   <cxx-Effects>Constructs a <code>future</code> object from the shared state referred to by
   <code>rhs</code>.
 
-  
+
 The <code>future</code> becomes ready when one of the following occurs:
 <ul>
   <li>
-    Both the outer and the inner <code>future</code>s are ready. The <code>future</code> inherits the value or the exception from the inner <code>future</code>.
+    Both the <code>rhs</code> and <code>rhs.get()</code> are ready. The value or the exception from <code>rhs.get()</code> is stored in the <code>future</code>'s shared state.
```
Comment (K-ballo) [inline future.html]: "the `rhs`" sounds weird.

Comment (viboes) [inline future.html]: Agreed. There is no rhs on a copy constructor.

**Comment by arturl** [comment]
It's done this way elsewhere, for example see 30.6.6 para 3:
```
future(const future& rhs) = delete;
```

**Comment by K-ballo** [comment]
I was referring merely to the **"the"** before `rhs`.

**Pull request merged**